src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +14,26 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String FIRST_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder fileContent = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            while (reader.ready()) {
+                fileContent.append(reader.readLine());
```

Comment:
it seems the lines are not separate from each other after you append them to StringBuilder

Reply:
done